sonos/__init__.py

```diff
@@ -343,7 +343,11 @@ def dispose(self):
             return
 
         for subscription in self._events:
-            subscription.unsubscribe()
+            try:
+                subscription.unsubscribe()
+            except:
+                continue
+
```

Comment:
In which case an exception is thrown by `unsubscribe()` method? Is it OK to simply ignore the exception or should it be logged so users can find problems with it?